demisto_sdk/commands/format/update_generic_yml.py

```diff
@@ -132,6 +133,9 @@ def update_tests(self) -> None:
             pack_path = os.path.dirname(os.path.dirname(os.path.abspath(self.source_file)))
             test_playbook_dir_path = os.path.join(pack_path, TEST_PLAYBOOKS_DIR)
             test_playbook_ids = []
+            file_entity_type = find_type(self.source_file, _dict=self.data, file_type='yml')
+            file_id = get_entity_id_by_entity_type(self.data, ENTITY_TYPE_TO_DIR.get(file_entity_type.value, ""))
+            commands, scripts = get_scripts_and_commands_from_yml_data(self.data, file_entity_type)
             try:
                 test_playbooks_files = os.listdir(test_playbook_dir_path)
```

Comment:
```suggestion
                test_playbooks_files = [tpb_file for tpb_file in listdir_fullpath(test_playbook_dir_path) if find_type(tpb_file) == FileType.TEST_PLAYBOOK)]
```
Lets reduce not needed nesting and code here (I cant suggest changes for code you did not write
- we can remove the if test_playbooks_files: (the for loop will not run if list is empty)
- After the above list comp filter the lines 143, 145 will not be needed
- the if in line 146 is not needed